platforms/opencl/src/OpenCLNonbondedUtilities.cpp

```diff
@@ -387,6 +394,11 @@ void OpenCLNonbondedUtilities::computeInteractions(int forceGroups, bool include
         return;
     KernelSet& kernels = groupKernels[forceGroups];
     if (kernels.hasForces) {
+        #if !(__APPLE__ && defined(__aarch64__))
```

Comment:
This check is unnecessary.  ARM Macs don't have AMD GPUs.

Reply:
Good to know. Check has been removed.